docker/Dockerfile.dev

```diff
@@ -11,13 +11,14 @@ RUN apt-get update --fix-missing
 RUN apt-get -y install curl
 RUN curl -sL https://deb.nodesource.com/setup_10.x | bash -
 RUN apt-get -y install nodejs python python-dev \
+    fish man \
```

Comment:
It's extremely critical that `fish man` is on its own line.

(`man` is needed for fish's auto-completion to function properly, as it contains `apropos`)